app/controllers/api/concepts/makers_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
           {
             avatar_url: user.avatar_url,
             handle: user.handle,
+            flair: user&.flair,
```

Comment:
In the line above we use `user.handle`, so if `user` is `nil` it would already have crashed

```suggestion
            flair: user.flair,
```